product_profile_example/profile.py

```diff
@@ -37,7 +20,7 @@ def _get_types(self):
     purchase_ok = fields.Boolean(
         string='Can be Purchased')
     available_in_pos = fields.Boolean()
-    route_ids = fields.Many2many(
+    profile_default_route_ids = fields.Many2many(
```

Comment:
By renaming a field, you breaks the API and potentially breaks modules based on this API